app/controllers/cim_instance_controller.rb

```diff
@@ -147,7 +147,7 @@ def process_show(associations = {})
     else
       whitelisted_key = associations.keys.find { |key| key == @display }
       if whitelisted_key.present?
-        model_name = whitelisted_key.singularize.classify.constantize
+        model_name = whitelisted_key.singularize.classify
         drop_breadcrumb( {:name=>@record.evm_display_name+" (All #{ui_lookup(:tables => @display.singularize)})", :url=>"/#{self.class.table_name}/show/#{@record.id}?display=#{@display}"} )
```

Comment:
@matthewd : I am not the one to ask here. I did not write that stuff, I am just trying to understand it :-(

Looking at the sources and at the `en.yml` it seems that both plural and singular values for both keys `:table` and also `:tables` may be used.

In this case the `:tables => foo.singularize` will result make the `Dictionary` look for the singular version into `en.yml` and then call `.pluralize` on the returned string.

I can just say that this really is as crazy as it sounds and we have an item here: https://github.com/ManageIQ/manageiq/issues/3775 to do something about it when we get back to i18n stuff. In 2nd or 3rd step we should replace all these calls with gettext calls.